scripts/update-web-workspace.sh

```diff
@@ -0,0 +1,5 @@
+echo Upgrading related Workspace to $1
+
+cd packages/playground && yarn add @appbaseio/reactivesearch@$1 && cd ../maps && yarn add @appbaseio/reactivesearch@$1 && cd ../..
```

Comment:
remove `yarn` from here too.
and you're not updating the demos inside `site` folder.
I think only one script can work with a recursive function
- Start from the root and traverse all the folder
- Check if the folder has package.json with reactivesearch dependency
- update it to the latest
- move forward and repeat it for all the nested folders until the whole directory is covered

Reply:
> and you're not updating the demos inside site folder.

For `site` we already have scripts & we need yarn there to update `yarn.lock` & create a build.

> remove yarn from here too.

We have yarn.lock file in playground as well. So we may need to use yarn their also.